<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>X2Focus</class>
 <widget class="QDialog" name="X2Focus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>620</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>464</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MoonLite NightCrawler Focuser</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="X2Form" name="X2Form">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>440</width>
       <height>460</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>610</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>100</width>
       <height>200</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>512</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6"/>
     </widget>
     <widget class="QPushButton" name="pushButtonOK">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>550</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>550</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_System">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>401</width>
        <height>451</height>
       </rect>
      </property>
      <property name="title">
       <string>System Settings</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Serial Number:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Current System Voltage:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>271</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Temperature offset adjustment (degrees C):</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_Brightness">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Display Active Brightness:</string>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>381</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_SleepBright">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Display Sleep Brightness:</string>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>381</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_SN">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>59</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_Voltage">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>60</y>
         <width>59</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_Temp">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>95</y>
         <width>67</width>
         <height>24</height>
        </rect>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>-12.500000000000000</double>
       </property>
       <property name="maximum">
        <double>12.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>381</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>*WARNING: This operation is very time consuming and cannot be interrupted. Homing is rarely needed, and manual homing is recommended.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>330</y>
         <width>131</width>
         <height>32</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Home Focuser*</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>370</y>
         <width>131</width>
         <height>32</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Home Rotator*</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>410</y>
         <width>131</width>
         <height>32</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Home Auxiliary*</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>230</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Invert (flip) display</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Disable encoder knobs</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_Focuser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>470</y>
        <width>401</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Focuser Settings</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>281</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Motor Step Delay (100 microseond intervals):</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_MotorStep">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>25</y>
         <width>71</width>
         <height>24</height>
        </rect>
       </property>
       <property name="minimum">
        <number>7</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>7</number>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>X2Form</class>
   <extends>QFrame</extends>
   <header>x2form.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>X2Form</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
